scripts/in_container/run_docs_build.sh

```diff
@@ -28,7 +28,7 @@ sudo rm -rf "${AIRFLOW_SOURCES}/docs/_api/*"
 
 sudo -E "${AIRFLOW_SOURCES}/docs/build_docs.py" "${@}"
 
-if [[ ${GITHUB_ACTIONS} == "true" && -d "${AIRFLOW_SOURCES}/docs/_build/html" ]]; then
+if [[ ${GITHUB_ACTIONS:="false"} == "true" && -d "${AIRFLOW_SOURCES}/docs/_build/html" ]]; then
     rm -rf "/files/documentation"
     cp -r "${AIRFLOW_SOURCES}/docs/_build/html" "/files/documentation"
```

Comment:
Should we move instead of copy? This is a lot of files and may take a moment.

Reply:
Nope. They are mounted from another filesystem (mounted from the host).. Moving will not be faster because it will effectively be doing copy + delete.

Reply:
Can you add a note in code about it?

Reply:
Is it really that important that you need a note about it? The whole point about having "/files" and that they are used for transferring the files is described already in the #10380.

https://github.com/PolideaInternal/airflow/blob/future-ci-documentation/CI.rst#ci-environment (see MOUNT_FILES) parameter.

Reply:
This would be a note for others who may wonder in the future why we are using cp and not mv. However, if this change is already merged then we don't have to.